File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-persons-vetting_result.ts


import { GetPersonsVetting_Item } from './get-persons-vetting_item';
import { IGetPersonsVetting_Result } from './iget-persons-vetting_result';

export class GetPersonsVetting_Result implements IGetPersonsVetting_Result {
  
	public VettingCollection?: GetPersonsVetting_Item[];
  
}

File: GTTS/INL.PersonService.Models/GetPersonsVetting_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public class GetPersonsVetting_Result : IGetPersonsVetting_Result
    {
        public List<GetPersonsVetting_Item> VettingCollection { get; set; }
    }
}
